vector/src/main/java/im/vector/app/features/spaces/explore/SpaceDirectoryController.kt

```diff
@@ -162,6 +169,28 @@ class SpaceDirectoryController @Inject constructor(
                     }
                 }
             }
+            if (hierarchySummary?.nextToken != null) {
+                val paginationStatus = data.paginationStatus[currentRootId] ?: Uninitialized
+                if (paginationStatus is Fail) {
+                    errorWithRetryItem {
+                        id("error_${currentRootId}_${hierarchySummary.nextToken}")
+                        text(host.errorFormatter.toHumanReadable(paginationStatus.error))
+                        listener { host.listener?.retry() }
```

Comment:
I think calling `retry()` is not correct here, you should call `host.listener?.loadAdditionalItemsIfNeeded()`